docs/.vitepress/components/demos/CustomTracking.vue

```diff
@@ -4,28 +4,27 @@
       Track function:
       <select v-model="selected">
         <option v-for="option in options" :key="option.text" :value="option">
-          {{ option.text }}
+          {{ option.text }}
         </option>
       </select>
     </p>
 
-    <qrcode-stream :key="_uid" :track="selected.value" @init="logErrors" />
+    <qrcode-stream :track="selected.value" @init="logErrors" />
```

Comment:
I don't quite remember what this was good for. But why did you remove it?

Reply:
Because it "[is reserved for internal use and it's important to keep it private (and not rely on it in user code) so that we keep the flexibility to change its behavior for potential future use cases](https://github.com/vuejs/vue/issues/5886#issuecomment-308625735)".

Reply:
Alright 👍 but can you guess what it’s purpose might have been? I think it had something to do with rendering

Reply:
Normally you'd like to use keys for lists, but I don't see a list being used here. So I can't guess why it was used here.